data/json/recipes/recipe_food.json

```diff
@@ -59,6 +59,42 @@
   ]
 },{
   "type" : "recipe",
+  "result": "salt",
+  "id_suffix": "from_mineral",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_OTHER",
+  "skill_used": "cooking",
+  "difficulty": 0,
```

Comment:
Using iodine to refine rock salt is not common knowledge.
Should either be limited to books or have a significantly higher difficulty.